games-puzzle/kpat/patches/kpat-17.12.1.patchset

```diff
@@ -0,0 +1,64 @@
+From 53df3fa42ad54fc0ee8933b89aa597e33f1f0b3d Mon Sep 17 00:00:00 2001
+From: Khaled Berraoui <khallebal@gmail.com>
+Date: Tue, 2 Jan 2018 21:32:27 +0100
+Subject: Disable Doctools and Dbus to fix start up crash
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8f738bf..588aff3 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -14,8 +14,8 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
+     ConfigWidgets
+     CoreAddons
+     Crash
+-    DBusAddons
+-    DocTools
++#	DBusAddons
++#	DocTools
```

Comment:
You could just remove them. The commit could easily be reverted if these KF5 libs are fixed

```diff
-+#	DBusAddons
-+#	DocTools